readme.txt

```diff
@@ -0,0 +1,27 @@
+=== Wikipedia Preview ===
+Contributors: Wikimedia Foundation
+Donate link: 
```

Comment:
Should we add a donate link? It's optional

https://donate.wikimedia.org/wiki/Ways_to_Give

Reply:
yes